fiftyone/operators/server.py

```diff
@@ -229,11 +230,47 @@ async def post(self, request: Request, data: dict) -> dict:
         return result.to_dict() if result else {}
 
 
+class SubscribeToExecutionStoreAsOperator(HTTPEndpoint):
```

Comment:
Shouldn't this just be a way to call an SseOperator... if one just happens to implement subscriptions great. Why are we tying the two concepts together?